amundsen_application/api/metadata/v0.py

```diff
@@ -80,8 +80,8 @@ def get_table_metadata() -> Response:
     """
     try:
         table_key = get_query_param(request.args, 'key')
-        list_item_index = get_query_param(request.args, 'index')
-        list_item_source = get_query_param(request.args, 'source')
+        list_item_index = request.args.get('index', None)
```

Comment:
What would be the case when either `index` or `source` null?

Reply:
In React FE `index` and `source` are optional parameters but in Flask they are required parameters. To get around this we added `index` and `source` with no value to the URL. E.g. `v0/table?key=hive...fact_driver&index=&source=` This ends up being read as empty string (vs None in new code).

I changed these to optional so React can make requests like `v0/table?key=hive...fact_driver` and not fail.